10x/hca-dcp/wrapper_10x_count.wdl

```diff
@@ -10,55 +10,27 @@ task GetInputs {
 
   command <<<
     python <<CODE
-    import json
-    import requests
-    import subprocess
-    import time
+    from secondary_analysis import utils
 
     # Get bundle manifest
     uuid = '${bundle_uuid}'
     version = '${bundle_version}'
+    dss_url = '${dss_url}'
```

Comment:
I can't decide if I like having these variable conversions or not. On one hand, it makes it easier if we want to change the name of the wdl variable. On the other hand, it makes it less clear in the code where the wdl variable is being used and adds lines. 

@dshiga I see your commit originally started doing these conversions. Interested in hearing why if you have time :)

Reply:
If I understand what you're asking, I did it this way because I needed to use the uuid and version in multiple places in the Python code and didn't want to do the conversion in multiple places. So I did it once at the beginning and assigned the results to Python variables.

I see what you mean, but I prefer doing it this way because it provides a cleaner handoff from WDL to Python code, as opposed to sprinkling conversions through more of the code. An even cleaner interface would be to make the Python invokable as a script and make all WDL variables command line parameters to that script. The downside there is that it's a bit more complicated, so I don't think it's really necessary here, though I like that pattern overall.

Reply:
Awesome, that's exactly what I wanted to learn. Thanks for the explanation @dshiga